File: js/src/aiSuggester.js
import OpenAI from "openai";
const client = new OpenAI({ apiKey: process.env.OPENAI_API_KEY });

export async function getAISuggestion(lintOutput) {
    const prompt = `Here is the linter output:\n${lintOutput}\nProvide actionable suggestions and reasoning.`;
    const response = await client.chat.completions.create({
        model: "gpt-4",
        messages: [{ role: "user", content: prompt }],
        max_tokens: 300
    });
    return response.choices[0].message.content;
}

File: python/ai_linter/ai_suggester.py
import openai
openai.api_key = "YOUR_OPENAI_API_KEY"

def get_ai_suggestion(lint_output: str) -> str:
    prompt = f"Here is the linter output:\n{lint_output}\nProvide actionable suggestions and reasoning."
    response = openai.Completion.create(
        model="gpt-4",
        prompt=prompt,
        max_tokens=300
    )
    return response.choices[0].text.strip()